config/opal_mca.m4

```diff
@@ -570,8 +571,8 @@ AC_DEFUN([MCA_CONFIGURE_NO_CONFIG_COMPONENT],[
 #                         dso_components_variable,
 #                         static_ltlibs_variable,
 #                         allowed_to_succeed,
-#                         [eval if should build],
-#                         [eval if should not build])
+#                         [set to 1 if should build],
+#                         [set to 0 if should not build])
```

Comment:
This isn't quite right -- the comment implies that there are 10 parameters.  Could you clarify?

It might also be worth mentioning in a comment for each of the macros why there are 9 params instead of 10 (because it's a departure from the m4/Open MPI convention of "eval this if true, eval that if false").

Reply:
Comment is dead wrong, will fix